joomla/joomla-cms
# [4.0] Update Bootstrap to 5.1.1

Pull Request for Issue # .

### Summary of Changes
- Update to 5.1.1
- refactor the root colours to use the custom ones
- remove the empty mixin file and all the calls to it

### Testing Instructions
Apply the patch
Run `npm ci`

Check that ALL 3 templates are ok, not broken **AND** customising the colours still works as expected

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required

NO

Some short explanations of what's happening here:
- We override the _root.scss of the bootstrap
- In the variables, we place all the Atum specific variables in an array `$atum-colors`
- The $body-color is removed from the variables as it's already converted to CSS VAR in the bootstrap source https://github.com/twbs/bootstrap/blob/1df098361cac04217d6a464c80e890c4335ecb5c/scss/_root.scss#L47 and we assign it to the Atum custom CSS Var
- The root loops the `$atum-colors` and appends the CSS VARS to the `:root`
- The mixin part was just a cleanup
